server/src/main/java/org/elasticsearch/action/admin/cluster/storedscripts/GetScriptContextResponse.java

```diff
@@ -122,11 +108,11 @@ public boolean equals(Object o) {
             return false;
         }
         GetScriptContextResponse that = (GetScriptContextResponse) o;
-        return contextNames.equals(that.contextNames);
+        return new HashSet<>(contexts).equals(new HashSet<>(that.contexts));
```

Comment:
Why are hashsets being created here? We shouldn't do any allocations in an equals method.

Reply:
Removed allocation.